spec/schema_version_spec.cr

```diff
@@ -0,0 +1,53 @@
+require "./spec_helper"
+require "../src/lavinmq/schema"
+
+describe LavinMQ::SchemaVersion do
+  describe "Schema version" do
+    it "Empty file should raise EmptyFile" do
+      with_amqp_server do |_s|
```

Comment:
do you need an amqp server for this spec?

Reply:
No, just need a data_dir. Changed to using `with_datadir `instead!